docs/netlify.toml

```diff
@@ -94,6 +94,11 @@
   to = "/latest/integrations/"
   force = true
 
+[[redirects]]
```

Comment:
I was wrong! This needs to go above the more general redirect on line 87, because the first match wins in this file.

Reply:
Oh thank you and in that case , I added it right above the first redirect that goes to /latest/integrations.